CMakeLists.txt

```diff
@@ -254,7 +276,7 @@ add_subdirectory(man)
 
 # special quirk for old kernel versions
 if (HW_BREAKPOINT_COMPAT)
-    target_compile_definitions(lo2s PRIVATE HW_BREAKPOINT_COMPAT)
+    set(lo2s PRIVATE HW_BREAKPOINT_COMPAT)
```

Comment:
These three `set` don't do what you think they are doing. And I don't even know, what they are doing.

Reply:
While you are on it, rename this to something like `USE/HAVE_HW_BREAKPOINT_COMPAT `